Firestore/src/Query.php

```diff
@@ -362,13 +362,17 @@ public function where($fieldPath, $operator, $value)
                 ]
             ];
         } else {
+            $encodedValue = ($operator == FieldFilterOperator::IN
```

Comment:
Always use strict comparison unless there's a strong reason not to.

```suggestion
            $encodedValue = ($operator === FieldFilterOperator::IN
```